src/steps/lido/lido-stake-shortcut-step.ts

```diff
@@ -0,0 +1,85 @@
+import { RelayAdaptContract } from '../../contract/adapt/relay-adapt-contract';
+import {
+    RecipeERC20AmountRecipient,
+    RecipeERC20Info,
+    StepConfig,
+    StepInput,
+    StepOutputERC20Amount,
+    UnvalidatedStepOutput,
+} from '../../models/export-models';
+import { compareERC20Info } from '../../utils/token';
+import { Step } from '../step';
+import { getBaseToken } from '../../utils/wrap-util';
+import { ContractTransaction } from 'ethers';
+import { LidoWSTETHContract } from '../../contract/lido';
+import { Provider } from 'ethers';
+
+export class LidoStakeShortcutStep extends Step {
+    readonly config: StepConfig = {
+        name: 'Lido Staking [wstETH]',
+        description: 'Stake ETH to get wstETH',
+    };
+
+    private provider: Provider;
+
+    constructor(private wstETHTokenInfo: RecipeERC20Info, provider: Provider) {
+        super();
+        this.provider = provider;
+    }
+
+    private async getWrappedAmount(stakeAmount: bigint): Promise<bigint> {
+        const wstETHContract = new LidoWSTETHContract(
+            this.wstETHTokenInfo.tokenAddress,
+            this.provider,
+        );
+        const wrappedAmount = await wstETHContract.getWstETHByStETH(stakeAmount);
+        return wrappedAmount;
+    }
```

Comment:
Does this works? I think so, can you check?
```suggestion
    private getWrappedAmount(stakeAmount: bigint): Promise<bigint> {
        const wstETHContract = new LidoWSTETHContract(
            this.wstETHTokenInfo.tokenAddress,
            this.provider,
        );
        return  wstETHContract.getWstETHByStETH(stakeAmount);
    }
```